include/i2c_io.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file    i2c_io.h
+ * @author  Robert Taylor
+ * @date    2021-08-04
+ */
+
+#include "FreeRTOS.h"
+#include "HL_i2c.h"
+#include "os_semphr.h"
+
+#define I2C_TIMEOUT_MS 100
```

Comment:
Curious how these values were chosen? Nothing wrong with them, just wondering if they are related to some other requirements?

Reply:
More or less at random. I think it's too big but we do have an i2c transfer that requests 100 bytes in another submodule. I don't want the software to timeout when things are actually going fine, just taking a long time

Reply:
Well, normally it should never happen and if it does, it probably indicates other issues.